src/clib/CMakeLists.txt

```diff
@@ -157,7 +154,12 @@ endif ()
 
 include(CheckTypeSize)
 check_type_size("size_t" SIZEOF_SIZE_T)
+MESSAGE("size_t ${SIZEOF_SIZE_T}")
 CHECK_TYPE_SIZE("long long" SIZEOF_LONG_LONG)
-if (NOT ${SIZEOF_SIZE_T} EQUAL ${SIZEOF_LONG_LONG})
+MESSAGE("long long ${SIZEOF_LONG_LONG}")
+if (SIZEOF_SIZE_T AND SIZEOF_LONG_LONG AND NOT ${SIZEOF_SIZE_T} EQUAL ${SIZEOF_LONG_LONG})
```

Comment:
But now is this just ignoring the comparison because one of these values does not get defined?

Seems like it was the changing of the flags that caused this test to return different results.

Reply:
Doesn't matter, it didn't work anyway.  It seems to have a problem with -std=c99 defined twice so I'm trying to clear the env vars defined for autoconf before running cmake.